src/app/layout/components/user-menu/user-menu.component.scss

```diff
@@ -0,0 +1,32 @@
+#card {
+ background: #4a61e0;
+ border-radius: 0px;
+ align-content: center;
+ display: flex;
+ flex-direction: column;
+ align-items: center;
+}
+#card h2, #card h4 {
+  color: white;
+  font-weight: normal;
+}
+#card h4 {
+  font-style: italic;
+  margin-top: 9px;
+}
+#card h2 {
+  margin-bottom: 9px;
+}
+#card img {
+  height: 32%;
+  width: 35%;
+  border-radius: 71px;
```

Comment:
do you want actual rounded image. If so than since height and with is in percentages, border radius in pixels can produce strange results. What I suggest is fixed height and width in pixels and possible same pixels amount (examples 70x70px) and border radius of 50%.